Gopkg.toml

```diff
@@ -0,0 +1,21 @@
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#  name = "github.com/x/y"
+#  version = "2.4.0"
+
```

Comment:
Add constraints on bot api and go-humanise

Reply:
It's already in lock file. https://github.com/aerokube/aerokubot/pull/6/commits/90868db40228c405dcfd3c73c7cc33cd417cecfe#diff-bd247e83efc3c45ae9e8c47233249f18R7

Reply:
I know. But in lock file there are concrete revisions. Mb its better to define it as restrictions?